docker/Dockerfile.nginx

```diff
@@ -7,12 +7,12 @@ WORKDIR /tmp
 RUN apk add git openssl-dev pcre-dev zlib-dev libc-dev gcc make
 RUN NGINX_VERSION=$(nginx -v 2>&1 | sed 's/nginx version: nginx\///') && \
     wget -qO- https://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz | tar xvz && \
-    mv nginx-${NGINX_VERSION} nginx
+    mv nginx-${NGINX_VERSION} chainstack
 COPY ./src /tmp/ngx_http_websocket_stat_module
-RUN cd ./nginx && \
+RUN cd ./chainstack && \
     ./configure  --with-compat \
     --add-dynamic-module=../ngx_http_websocket_stat_module && \
     make modules
 
-FROM k8s.gcr.io/ingress-nginx/controller:v0.44.0
-COPY --from=builder /tmp/nginx/objs/ngx_http_websocket_stat_module.so /etc/nginx/modules/
+FROM k8s.gcr.io/ingress-nginx/controller:v1.8.1
```

Comment:
I used v1.8.2 , any chance that's why half of the fmt directives do not work for me?  if I add any of the 'remote_addr' / 'server_addr' / 'uri' / 'request' / etc'   the nginx will crash.  what is the version to use, is 1.8.1 the best option to make it work? i thought it'll be compatible with a few other nginx versions but seems like there are major differences 😅 to the point that some log parts won't work at all.   
I managed to make my log work with those settings 
```
    ws_log_format open "OPEN, SOURCE: $ws_packet_source (CODE $ws_opcode)";
    ws_log_format server "SERVER->CLIENT: $time_local | SOURCE: $ws_packet_source AGE: $ws_conn_age seconds, Size: $ws_message_size (CODE $ws_opcode)";
    ws_log_format client "CLIENT->SERVER: $time_local | SOURCE: $ws_packet_source AGE: $ws_conn_age seconds, Size: $ws_message_size (CODE $ws_opcode)";
    ws_log_format close "CLOSE, SOURCE: $ws_packet_source AGE: $ws_conn_age seconds (CODE $ws_opcode)";
```

but if i'll add any other directive from the available ones, it'll crash 🙈

Reply:
btw thanks for maintaining this module, i'm baffled that there's no native support in nginx to debug websockets disconnets/issues 😵

Reply:
@zaslavskii this question is for you

Reply:
@Dmitry1987 thanks for the question! Let me take a look at the issue closer and get back to you